openpype/tools/utils/delegates.py

```diff
@@ -291,3 +291,30 @@ class PrettyTimeDelegate(QtWidgets.QStyledItemDelegate):
     def displayText(self, value, locale):
         if value is not None:
             return pretty_timestamp(value)
+
+
+class LoadedInSceneDelegate(QtWidgets.QStyledItemDelegate):
```

Comment:
No - pretty tightly coupled currently. I intended originally to generalize it so it was a delegate that would allow a mapping of value to str and color, e.g. but felt that there was no need currently to make it more complex than needed.

Like this? https://github.com/pypeclub/OpenPype/pull/3649/commits/d5cb828edc611f9ed9f98b61ae5ae3591d797640